11-query/04-delete-query.md

```diff
@@ -5,30 +5,32 @@ longTailKeywords: grakn delete data, graql delete query, graql delete instances
 Summary: Delete queries in Grakn.
 ---
 
-## Delete Instances of an Entity Type
-To delete an instance of an entity type from the knowledge graph, we use a [match clause](../11-query/01-match-clause.md) followed by the `delete` keyword and the variable to be deleted. To try the following examples with one of the Grakn clients, follows these [Clients Guide](#clients-guide).
+## Delete Instances of a Type
+To delete an instance of a any type from the knowledge graph, we use a [match clause](../11-query/01-match-clause.md) followed by the `delete` keyword and statements indicating data to delete.
```

Comment:
_instance of a ~~any~~ type_
and **the** statements indicating data to delete

Reply:
I don't think an extra `the` makes grammatical sense?